docs/release-notes/NuGet-6.5.md

```diff
@@ -0,0 +1,125 @@
+---
+title: NuGet 6.5 Release Notes
+description: Release notes for NuGet 6.5 including new features, bug fixes, and DCRs.
+author: <TODO: GitHubAlias>
+ms.author: <TODO: MicrosoftAlias>
+ms.date: 2/10/2023
+ms.topic: conceptual
+---
+
+# NuGet 6.5 Release Notes
+
+NuGet distribution vehicles:
+
+| NuGet version | Available in Visual Studio version | Available in .NET SDK(s) |
+|:---|:---|:---|
+| [**6.5**](https://nuget.org/downloads) | [Visual Studio 2022 version 17.5](https://visualstudio.microsoft.com/downloads/) | [<TODO: Full SDK Version>](https://dotnet.microsoft.com/download/dotnet-core/<TODO: SDKMajorMinorVersionOnly)<sup>1</sup> |
+
+<sup>1</sup> Installed with Visual Studio 2022 with.NET Core workload
+
+## Summary: What's New in 6.5
+
+* [Feature]: add or update package versions in the Directory.Packages.props file for CPM projects - [#11890](https://github.com/NuGet/Home/issues/11890)
+
+* Package Source Mapping UI - Allow the Creation/Removal of package source mappings in the NuGet Options UI - [#11363](https://github.com/NuGet/Home/issues/11363)
+
+* Package Source Mapping UI - List Package Source Mappings in the NuGet Options UI - [#11362](https://github.com/NuGet/Home/issues/11362)
+
+* dotnet build run into 429 and ignore Retry-After Header - [#10558](https://github.com/NuGet/Home/issues/10558)
```

Comment:
Maybe change this to the PR name? 

> Observe Retry-After delay on HTTP request retry 

@zivkan any suggestions?

Reply:
The PR used the title "Observe Retry-After delay on HTTP request retry".  I think that's a good title. Or maybe "Observe Retry-After header delay value on HTTP request retry